__tests__/config/formats/typescript-export-definition.test.ts

```diff
@@ -0,0 +1,124 @@
+import {typescriptExportDefinition} from '../../../config/formats/typescript-export-defition'
```

Comment:
Heads up, typo in the original filename. 

```suggestion
import {typescriptExportDefinition} from '../../../config/formats/typescript-export-definition'
```